algorithms/ar-bsrt/typescript/bubble_sort.ts

```diff
@@ -0,0 +1,27 @@
+//bubbleSort function
+function bubbleSort (arr : number[]) {
+    let len : number = arr.length;
+    let swapped : boolean;
+    do {
+        swapped = false;
+        for (let i = 0; i < len; i++) {
+            if (arr[i] > arr[i + 1]) {
+                let tmp = arr[i];
+                arr[i] = arr[i + 1];
+                arr[i + 1] = tmp;
+                swapped = true;
+            }
+        }
+    } while (swapped);
+    return arr;
+}
+
+//Function to test bubbleSort
+let main = () => {
+	let arr = [7, 9, 5, 8, 2, 3, 1, 6, 4];
+    console.log(bubbleSort(arr));
```

Comment:
Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6329238)

Reply:
console.log line has been replaced by simply calling the function and passing the array, please check again :)